msi/release.sh

```diff
@@ -118,10 +118,35 @@ SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_PATH" ||
 die "Could not switch directory to $SCRIPT_PATH"
 
+# Generate the ReleaseNotes.html file
+test -f ../ReleaseNotes.html &&
+test ../ReleaseNotes.html -nt ../ReleaseNotes.md &&
+test ../ReleaseNotes.html -nt release.sh || {
+    # Install markdown
+    type markdown ||
+    pacman -Sy --noconfirm markdown ||
+    die "Could not install markdown"
+
+    (printf '%s\n%s\n%s\n%s %s\n%s %s\n%s\n%s\n%s\n' \
+        '<!DOCTYPE html>' \
+        '<html>' \
+        '<head>' \
+        '<meta http-equiv="Content-Type" content="text/html;' \
+        'charset=UTF-8">' \
+        '<link rel="stylesheet"' \
+        ' href="usr/share/git/ReleaseNotes.css">' \
+        '</head>' \
+        '<body class="details">' \
+        '<div class="content">'
+     markdown ../ReleaseNotes.md ||
+     die "Could not generate ReleaseNotes.html"
+     printf '</div>\n</body>\n</html>\n') >../ReleaseNotes.html
+}
+
```

Comment:
Instead of duplicating this code, it would be better to refactor it into, say, `render-release-notes.sh` and call that script both from `installer/release.sh` and `msi/release.sh`. As a bonus, this script could have an option to expect the `.css` in a different location than `./usr/share/git/`.